src/Microsoft.OData.Edm/Csdl/CsdlReader.cs

```diff
@@ -340,6 +340,21 @@ private bool TryParse(IEnumerable<IEdmModel> referencedModels, bool includeDefau
                     {
                         model.SetEdmVersion(edmVersion);
                     }
+
+                    // Note: Considered manually replacing expressions after adding the terms to the model
+                    // so I could initialize them with default values. (Would need to replace because
+                    // the annotation.Value property is readonly.) However, this is somewhat expensive/complex.
+                    /*IEnumerable<IEdmVocabularyAnnotation> annotations = model.VocabularyAnnotations;
+                    for (int i = 0; i < annotations.Count(); i++)
+                    {
+                        IEdmVocabularyAnnotation currAnnotation = annotations.ElementAt(i);
+                        if (currAnnotation.UsesDefault)
+                        {
+                            IEdmVocabularyAnnotation replacementValue = new EdmExpression(...);
+                            IEdmVocabularyAnnotation replacementAnnotation = new IEdmExpression(...);
+                            annotations.SetElement(i, replacementAnnotation);
+                        }
+                    }*/

```

Comment:
Should remove this since it's not part of the final solution.

Reply:
Good catch, thanks! ✅